### PR TITLE
Makes roundending significantly harder by making the brain (Almost) indestructible

### PR DESCRIPTION
**Pull request by Majkl-J**
## About The Pull Request

See name. The only way to completely glass a brain now is cremation and the gibber (Soon to be changed probably hopefully)

## Why It's Good For The Game

Pretty sure we are not meant to roundend people. But there's so many hazards that might just end up removing you from the game, namely stuff like bombs gibbing you so hard your brain gets dusted in the explosion (Might be a synth only issue, to be checked). This makes it need much more deliberate effort to actually remove someone out of the game permanently.

## Changelog

:cl:
tweak: Brains are no longer easily destructible with fire, explosives, or acid. The gibber and crematorium still work
/:cl:

### CODE REVIEW

**Comment by Majkl-J** [comment]
This might not resolve the actual issue that made me create this, but I still believe this is a step in the right direction

**Comment by Cyprex** [comment]
what about deep frying?

**Comment by Cyprex** [comment]
Also selling via the cargo shuttle- anything that can destroy the blackbox(traitor objective) can probably destroy brains too

File: modular_zubbers/code/modules/surgery/organs/internal/lungs/_brain.dm [hunk 1]
```diff
@@ -0,0 +1,2 @@
+/obj/item/organ/internal/brain
+	resistance_flags = INDESTRUCTIBLE
```
Comment (Cyprex) [inline modular_zubbers/code/modules/surgery/organs/internal/lungs/_brain.dm:2]: this overrides the ones defined originally- are any defined? Not sure if it should be added in New()... or not.

Comment (Majkl-J) [reply modular_zubbers/code/modules/surgery/organs/internal/lungs/_brain.dm:2]: This flag is above all the others, in my testing this includes lava and acid resistance. And there are none

**Comment by Majkl-J** [comment]
> what about deep frying?

This is to mainly stop hazards like explosions and other stuff that can happen without deliberation. Deliberate RR is still possible, as it probably should be considering stuff like changelings exist.

**Pull request merged**